### ISSUE TITLE
Allow triggering tablet UI in apps by changing reported smallest width value per app (similar to dpi)

### ISSUE THREAD

**Issue by MmyGgithubAaccount** [ikws4/WeiJu#5]
Google recommends apps check [smallest width](https://developer.android.com/training/multiscreen/screensizes#TaskUseSWQuali) to trigger tablet UI. Smallest width ≥ 600 dp will trigger tablet UI in supported apps.

Smallest width is related to dpi according to the following formula
Smallest width = resolution × 160/dpi
So, for a 1080p screen, 600dp smallest width corresponds to a dpi of 288

However, just changing dpi via your app while having phone smallest width below 600 dp doesn't trigger tablet ui since apps check smallest width and not dpi to trigger tablet mode.

A solution would be to allow users to change app scaling using either dpi (as is already present) _or_ smallest width. That way, users like me that want tablet mode can use smallest width to trigger it while users who don't want tablet mode but still want low dpi can use the dpi setting

**Comment by LeeBinder** [comment]
hey @ikws4 , thanks for WeiJu! I'm also interested in tablet UI mode, but apart from that (did not want to open an "issue" for that): how good is your English on a scale 0-5? Do you understand inquiries like that from MmyGgithubAaccount ?

**Comment by ikws4** [comment]
> hey @ikws4 , thanks for WeiJu! I'm also interested in tablet UI mode, but apart from that (did not want to open an "issue" for that): how good is your English on a scale 0-5? Do you understand inquiries like that from MmyGgithubAaccount ?

Yes, I am understand what his inquiry, But the table UI is a bit troublesome to implement, I'm still thinking how to implement this.

**Comment by LeeBinder** [comment]
OK.

I recommend in Settings you change every "Open" (that's not a good translation at all for what each slider switch does) with "Toggle on/off".
